linux_os/guide/services/dns/disabling_dns_server/package_bind_removed/tests/test_config.yml

```diff
@@ -0,0 +1,3 @@
+deny_templated_scenarios:
+  - package-installed-removed.pass.sh
```

Comment:
You have denied tests for all products but only replaced for RHEL 9, should add tests the other products?